docs/react-hook-form.md

```diff
@@ -1,6 +1,6 @@
 ## ⚡ Forms handling
 
-We are using two libraries to manage forms in our app: [react-hook-form](https://github.com/react-hook-form/react-hook-form) and [zod](https://github.com/colinhacks/zod). The former is a library dedicated to form handling while the latter is a schema declaration and validation library.
+We are using two libraries to manage forms in our app: [react-hook-form](https://github.com/react-hook-form/react-hook-form) and [zod](https://github.com/colinhacks/zod). The former is a library dedicated to form handling while the latter is a schema declaration and parsing library.
```

Comment:
Might seem to be pendatic, but zod is more of a parsing library than a validation library. The distinction  being that with `zod` we are focusing in trying to coerce as much as possible an `unknown` onto a known type, mutating it if necessary.